docs/release/cut-sw-release.md

```diff
@@ -157,7 +157,8 @@ cd release-tools
 ./1-client-tarballs $NON_UPCOMING_VERSIONS
 ```
 
-You should get up to 8 tarballs for each version (excluding upcoming), 25-55 megs each and they should all have the version number in the name.
+You should get 7 tarballs.
+They should all have the version number in the name.
```

Comment:
These sentences seem superfluous if we're automatically testing the existence of the 7 tarballs below.